### ISSUE TITLE
Pins board function argument missing

### ISSUE THREAD

**Issue by gsingh91** [tidymodels/vetiver-python#33]
**Describe the bug**
The `.py` file generated after using function does not have` allow_pickle_read=True` option set, which is needed to write models as pins

**To Reproduce**
Run `vetiver.vetiver_write_app` function to generate the file

```
vetiver_write_app(
        board,
        "username/modelname",
        file="superbowlads.py"
)
```

**Expected behavior**
`allow_pickle_read=True` should be set by default

**Desktop (please complete the following information):**

- OS: Ubuntu
- Browser: Chrome
- Version: 0.1.3

CC: @xuf12

**Comment by isabelizimm** [comment]
This part of the app file is coming from `[board_deparse()](https://github.com/rstudio/pins-python/blob/c97b2a49da2f7a84e48f57faf6f739e64ef8b1fd/pins/constructors.py#L17)` function in `pins`. I wonder if this is a vetiver-specific issue, or makes sense to be part of pins? WDYT @machow? I'm happy to help with a PR in pins if that seems appropriate!

**Comment by machow** [comment]
Ah, thanks for catching--that makes to fix this in pins! I opened an issue to track!:  https://github.com/rstudio/pins-python#123

**Comment by isabelizimm** [comment]
Merged in `pins-python`, so closing here.

**Comment by machow** [comment]
released to pypi as pins v0.2.1!
